Kernel/Graphics/FramebufferDevice.cpp

```diff
@@ -120,6 +115,10 @@ UNMAP_AFTER_INIT FramebufferDevice::FramebufferDevice(const GraphicsDevice& adap
     , m_output_port_index(output_port_index)
     , m_graphics_adapter(adapter)
 {
+    auto name = String::formatted("fb{}", minor());
+    m_determined_name = KString::try_create(name.substring_view(0));
```

Comment:
This is a very clunky pattern, and like @IdanHo says, we should have a `KString::try_formatted()` or similar instead. :)

Reply:
I agree, maybe we could leverage this change to implement a `KStringBuilder` then, to have a complete isolation from `AK::String` patterns.